.github/workflows/workflow.yml

```
@@ -0,0 +1,50 @@
+
+  deploy:
+    runs-on: ubuntu-latest
+    needs: build # buildジョブが成功した後に実行
+    steps:
+      - name: Download build artifacts
+        uses: actions/download-artifact@v3
+        with:
+          name: dist
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
```

actionlint (1.7.4)

the runner of "aws-actions/configure-aws-credentials@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)